pkg/kubernetes/dynamicInformers.go

```diff
@@ -80,29 +79,12 @@ func newDynamicInformerFactory(ctx context.Context, cfg *restclient.Config, name
 // GetDynamicInformer returns informer for specified CRD group, version, name and namespace
```

Comment:
Update the function doc that the method accepts config

Reply:
done!